opm/material/fluidmatrixinteractions/EclDefaultMaterialParams.hpp

```diff
@@ -127,6 +127,13 @@ class EclDefaultMaterialParams : public EnsureFinalized
     bool inconsistentHysteresisUpdate() const
     { return true; }
 
+    template<class Serializer>
+    void serializeOp(Serializer& serializer)
+    {
+        serializer(*gasOilParams_);
```

Comment:
Not serializing `Swl_` is probably correct in the sense that it will be initialized in other ways, whereas the params members may contain mutating data (we already had this discussion about other classes). I would just say that it actually points out quite precisely what data should be moved *out of* these classes (and into some more obvious container of state data) eventually!

The Stone* classes contain the same logic as this one, so the comment above applies there as well (they contain local members that probably need not be serialized here).

It does however make we want to change the name of this class to something like DefaultThreephaseApproachMaterialParams, just "Default" is a bit misleading when it is really only the default way to make two two-phase tables into a three phase table. (And the Stone* classes are other ways of doing that.)